### DISCUSSION TITLE
How to use oracledb6 to connect to oracle11g on a mac arm chip computer?

### DISCUSSION THREAD

**Discussion by hanbingzi** [oracle/node-oracledb#1660] [Q&A]
Can I use mac arm chip computers to connect to oracle11g after oracledb6？

**Comment by sharadraju** [comment]
@hanbingzi You can use node-oracledb 6 [Thick mode ](https://node-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#enabling-node-oracledb-thick-mode)on M1/M2/M3 via Apple's Rosetta. Node-oracledb's default Thin mode does not support Oracle 11g.

Please follow the instructions in the blog [How to Install node-oracledb 5.5 and Oracle Database on Apple M1/M2 Silicon](https://medium.com/oracledevs/how-to-install-node-oracledb-5-5-and-oracle-database-on-apple-m1-m2-silicon-941fccda692f).

**Comment by odnodn** [comment]
late UPDATE for those trying to use node-oracledb with thick client on apple sillicone architectures (ARM64): since node-oracledb 6.6 macOS ARM64 (included binaries) is supported and an ARM64 instant client binary exists. The documentation still seems to be incomplete, I created an issue for this.
